README.md

```diff
@@ -1 +1,150 @@
-# budget-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Budget App</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Budget App ](#-budget-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [🚀 Deployment ](#-deployment-)
+  - [🎞️ Presentation Video ](#️-presentation-video-)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Budget App <a name="about"></a>
+
+The **Budget App** is an application for keeping track of our expenses in different categories.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- RUBY
+- Rails
+- Postgresql
+- HTML
+- CSS
+- Javascript
+
+### Key Features <a name="key-features"></a>
+
+- [ ] **Login**
+- [ ] **Sign Up**
+- [ ] **Logout**
+- [ ] **Create New category**
+- [ ] **Create New transaction**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- git
+- Ruby
+- Rails
+- Bundle gem
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:ybadr99/rails-budget-app.git
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails s
+```
+
+### Run tests
+
+To run tests, run the following commands:
+
+```sh
+  rubocop
+  rspec spec/
+  npm run stylelint
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Deployment <a name="Deployment"></a>
+
+Deployment: [linkw](https://budget-app-lhsh.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🎞️ Presentation Video <a name="video"></a>
+
+Video: [link](https://www.loom.com/share/991a3985fba44c2cb665a4031f446450)
+
```

Comment:
- [ ] The video should contain a highlight about a piece of the code that you find interesting or is proud of making.